src/TinyHelpers.AspNetCore/Attributes/ContentTypeAttribute.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using Microsoft.AspNetCore.Http;
+
+namespace TinyHelpers.AspNetCore.Attributes;
+
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+public class ContentTypeAttribute : ValidationAttribute
+{
+    private readonly string[] validContentType;
+    private readonly string[] imageContentTypes = new string[] { "image/jpeg", "image/png", "image/gif" };
+    private readonly string[] videoContentTypes = new string[] { "video/mp4", "video/ogg", "video/quicktime" };
```

Comment:
Se vuoi fornire una lista precompilata di content-type relativi ai formati video, questa lista dovrebbe essere il più possibile completa. Ad esempio, in questa lista manca **video/mpeg**. Dai un'occhiata a questo post: https://www.thoughtco.com/mime-types-by-content-type-3469108#toc-video-files-and-their-mime-types